rules/falco_rules.yaml

```diff
@@ -1172,6 +1172,23 @@
 - macro: calico_node
   condition: (container.image.repository endswith calico/node and proc.name=calico-node)
 
+# Return always_true from this macro to get warnings about the cups daemons
+# regular updates in /etc/cups/.
+- macro: cupsd_state_updates
+  condition: never_true
+
+- macro: cupsd_state_writing
+  condition: >
+    proc.name = cupsd and
+    fd.name in (
+       /etc/cups/subscriptions.conf,
+       /etc/cups/subscriptions.conf.N,
+       /etc/cups/subscriptions.conf.O,
+       /etc/cups/printers.conf,
+       /etc/cups/printers.conf.N,
+       /etc/cups/printers.conf.O
```

Comment:
Hello @petterreinholdtsen, I am not familiar with cupsd but just wanted to ask a question, are we sure that these are the only file it is writing? After installation, I am seeing in `/etc/cups` that there are also other files and directories. 
Maybe we can con consider using `startswith /etc/cups` instead of `in`. Just a question, in any case sgtm